<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Make trainData for diffLayer lineDetector</string>
  </property>
  <widget class="QGraphicsView" name="view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>204</width>
     <height>204</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewLarge">
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>10</y>
     <width>804</width>
     <height>804</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="view1">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>10</y>
     <width>204</width>
     <height>204</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>630</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTrainFileName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>630</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AA/AA</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sboxTrainFileName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>630</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>630</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="view2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>218</y>
     <width>404</width>
     <height>404</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBatchMonoCalcLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>660</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>BatchMonoCalcLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTraining">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>740</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Training</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chBoxDrawPredict">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>780</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DrawPredict</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPatch">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>660</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Patch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBatchGrayDirectLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BatchGrayDirectLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>158</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnCircle">
      <property name="text">
       <string>DrawCircle</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnLine">
      <property name="text">
       <string>DrawLine</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnRandDraw">
      <property name="text">
       <string>RandDraw</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnCalcTrainLabel">
      <property name="text">
       <string>CalcLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>700</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Batch on</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edPath">
      <property name="text">
       <string>All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnMergeData">
      <property name="text">
       <string>MergeData</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>741</y>
     <width>73</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="comboClassNo">
      <property name="maxCount">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Classes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>splitter</zorder>
  <zorder>view</zorder>
  <zorder>viewLarge</zorder>
  <zorder>view1</zorder>
  <zorder>btnSave</zorder>
  <zorder>leTrainFileName</zorder>
  <zorder>sboxTrainFileName</zorder>
  <zorder>btnLoad</zorder>
  <zorder>view2</zorder>
  <zorder>btnBatchMonoCalcLabel</zorder>
  <zorder>edPath</zorder>
  <zorder>btnTraining</zorder>
  <zorder>chBoxDrawPredict</zorder>
  <zorder>btnPatch</zorder>
  <zorder>comboClassNo</zorder>
  <zorder>label</zorder>
  <zorder>btnClear</zorder>
  <zorder>btnBatchGrayDirectLabel</zorder>
 </widget>
 <tabstops>
  <tabstop>btnLine</tabstop>
  <tabstop>view</tabstop>
  <tabstop>viewLarge</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doSaveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoad</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doLoadData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBatchMonoCalcLabel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doBatchMonoCalcLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>745</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTraining</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doTraining()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>815</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMergeData</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doMergeData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>741</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPatch</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doPatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>721</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>clearWorld()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>977</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalcTrainLabel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>calcTrainLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRandDraw</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doRandDraw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>781</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLine</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCircle</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doCircle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>681</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBatchGrayDirectLabel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doBatchGrayDirectLabel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>745</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doLine()</slot>
  <slot>clearWorld()</slot>
  <slot>calcTrainLabel()</slot>
  <slot>doCircle()</slot>
  <slot>doTraining()</slot>
  <slot>doMergeData()</slot>
  <slot>doLoadData()</slot>
  <slot>doRandDraw()</slot>
  <slot>doSaveData()</slot>
  <slot>doPatch()</slot>
  <slot>doBatchMonoCalcLabel()</slot>
  <slot>doBatchGrayDirectLabel()</slot>
 </slots>
</ui>
